distributed/worker.py

```diff
@@ -3718,6 +3718,11 @@ def convert_kwargs_to_str(kwargs, max_len=None):
         return "{{{}}}".format(", ".join(strs))
 
 
+# Keep `weight` for backwards compatibility
+def weight(k, v):
+    return sizeof(v)
```

Comment:
Maybe raise a deprecation warning and remove it in X releases? Do we have a process for smth like this?

Reply:
Good idea. We don't have a formal process for deprecation cycles today. This is definitely an area in which we can improve. I pushed a commit which adds a `deprecated` decorator which will hopefully make future deprecation cycles easier to manage.